src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```
@@ -1291,18 +1318,19 @@ protected function afterDeploymentSuccess(
         }
     }
 
-    protected function getRuntime(Document $resource, string $version): array
+    /**
+     * @return array|null Runtime config array, or null if the runtime is not supported (caller should fail deployment for functions).
+     */
+    protected function getRuntime(Document $resource, string $version): ?array
     {
         $runtimes = Config::getParam($version === 'v2' ? 'runtimes-v2' : 'runtimes', []);
         $key = $resource->getAttribute('runtime');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)